<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="userft-PC" name="Test2" tests="1" failures="1" timestamp="11 Nov 2017 07:28:45 GMT" time="8.683" errors="0">
  <testcase name="f" time="8.681" classname="sat_4_11.Assert_Validation">
    <failure type="java.lang.AssertionError" message="expected [OrangeHRMfggh] but found [OrangeHRM]">
      <![CDATA[java.lang.AssertionError: expected [OrangeHRMfggh] but found [OrangeHRM]
	at sat_4_11.Assert_Validation.f(Assert_Validation.java:31)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- f -->
</testsuite> <!-- Test2 -->
